doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -251,6 +251,12 @@ nRF Desktop
     The option was enabled by default, because the clock control is enabled by the Multiprotocol Service Layer (MPSL).
     Bluetooth stack with the SoftDevice Link Layer requires high frequency in GD to make the L2 cache and MRAM run fast to avoid code execution delays.
     The :ref:`nrf_desktop_dvfs` running on the application core selects the clock control and enables an automatic low GD frequency request (:kconfig:option:`CONFIG_CLOCK_CONTROL_NRF2_GLOBAL_HSFLL_REQ_LOW_FREQ`), so the application core would not request high frequency in the GD.
+  * Application configurations by increasing the following thread stack sizes to prevent stack overflows:
+
+    * The system workqueue thread stack (:kconfig:option:`CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE`).
+    * The CAF settings loader thread stack (:kconfig:option:`CONFIG_CAF_SETTINGS_LOADER_THREAD_STACK_SIZE`).
+
+    This change results from the Bluetooth subsystem transition to the PSA cryptographic API.
```

Comment:
Can we inform that we need bigger stack for the GATT database hash calculation that happens during the settings load operation? Consider also mentioning that the change affects only HID peripherals

Reply:
```suggestion
  * Application configurations for HID peripherals by increasing the following thread stack sizes to prevent stack overflows during the :c:func:`settings_load` operation:

    * The system workqueue thread stack (:kconfig:option:`CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE`).
    * The CAF settings loader thread stack (:kconfig:option:`CONFIG_CAF_SETTINGS_LOADER_THREAD_STACK_SIZE`).

    This change results from the Bluetooth subsystem transition to the PSA cryptographic API.
    The GATT database hash calculation now requires bigger stack size.
```